org.mixedrealitytoolkit.uxcomponents/CHANGELOG.md

```diff
@@ -3,13 +3,15 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 ## [3.2.0-development] - 2024-03-20
+## [3.3.0-development] - 2024-03-22
```

Comment:
The new header should be above the old one. And now bullet point under that.

Change to this:

```
## [3.3.0-development] - 2024-03-22

* Added unified font atlas and updated corresponding fonts and their materials. [PR #700](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/700)

## [3.2.0] - 2024-03-20

### Added

* Added Empty, Action, Action with Checkbox, and ToggleSwitch Experimental buttons prefabs [PR #649](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/649)
* Added corresponding MenuItems for use in Editor Tools [PR #649](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/649)
* Added CanvasUITearsheetDynamicFrontplate scene [PR #649](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/649)
* Added SimpleButton + demo scene [PR #635](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/635)

### Fixed

* Fixed support for UPM package publishing in the Unity Asset Store. [PR #519](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/519)
* Fixed Bug #643 - Experimental SimpleEmptyButton and SimpleActionButton prefabs have missing "See It Say It Label" GameObject reference [PR #646](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/646)
* Auto disable button's colliders when `StatefulInteractable` is disabled [PR #626](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/626)
```

Reply:
Fixed in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/700/commits/7a8b9eeb63b173779cbfe16c1c49e492c6093e70 and https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/700/commits/a6ba8d191dcf8753877fdf8bb030c0cf6503d457, resolving this.